x-pack/plugins/transform/public/app/sections/create_transform/components/step_details/common.ts

```diff
@@ -5,5 +5,96 @@
  * 2.0.
  */
 
+import type { TransformId, TransformPivotConfig } from '../../../../../../common/types/transform';
+
 export type EsIndexName = string;
 export type IndexPatternTitle = string;
+
+export interface StepDetailsExposedState {
+  continuousModeDateField: string;
+  continuousModeDelay: string;
+  createIndexPattern: boolean;
+  destinationIndex: EsIndexName;
+  isContinuousModeEnabled: boolean;
+  isRetentionPolicyEnabled: boolean;
+  retentionPolicyDateField: string;
+  retentionPolicyMaxAge: string;
+  touched: boolean;
+  transformId: TransformId;
+  transformDescription: string;
+  transformFrequency: string;
+  transformSettingsMaxPageSearchSize: number;
+  transformSettingsDocsPerSecond?: number;
+  valid: boolean;
+  indexPatternTimeField?: string | undefined;
+}
+
+const defaultContinuousModeDelay = '60s';
+const defaultRetentionPolicyMaxAge = '60s';
```

Comment:
`60s` seems very low to use as a default.

Reply:
As discussed - no default should be supplied, but the field should be obligatory if the Retention policy is switched on.

Reply:
Fixed in 20e47d2.